examples/android_app/app/src/main/java/com/example/llamastackandroiddemo/ExampleLlamaStackLocalInference.kt

```diff
@@ -96,10 +135,22 @@ class ExampleLlamaStackLocalInference(
                 )
                 .build()
         )
-        response = result.asChatCompletionResponse().completionMessage().content().string();
+        response = result.asChatCompletionResponse().completionMessage().content().string().toString();
```

Comment:
Converting to string and checking empty seems like the wrong way to check for tool calls – aren't there fields on completionMessage that are better ways to introspect this?

Reply:
checking to see if response is empty is the first check. After this we see if the toolCalls() in completionMessage() is populate and if so only then will we parse the toolCalls. I don't see any other familiar fields that indicate tool call otherwise (please let me know if there is and happy to take a look for change).

We do similar in the remote inferencing as well.

Reply:
@dltn  We see that a signal for whether is toolCall or not is present in streaming for events but CompletionMessage from ChatCompletionResponse does not have this at the moment.